config/betterquesting/DefaultQuests.json

```diff
@@ -306452,6 +306452,415 @@
+        }
+      },
+      "tasks:9": {
+        "0:10": {
+          "partialMatch:1": 1,
+          "autoConsume:1": 0,
+          "groupDetect:1": 0,
+          "ignoreNBT:1": 0,
+          "index:3": 0,
+          "consume:1": 0,
+          "requiredItems:9": {
+            "0:10": {
+              "id:8": "TConstruct:jerky",
+              "Count:3": 4,
+              "Damage:2": 5,
+              "OreDict:8": ""
+            }
+          },
+          "taskID:8": "bq_standard:retrieval"
+        }
+      },
+      "rewards:9": {
+        "0:10": {
+          "rewardID:8": "bq_standard:choice",
+          "index:3": 0,
+          "choices:9": {
+            "0:10": {
+              "id:8": "TConstruct:jerky",
+              "Count:3": 8,
+              "Damage:2": 0,
+              "OreDict:8": ""
+            },
+            "1:10": {
+              "id:8": "TConstruct:jerky",
+              "Count:3": 8,
+              "Damage:2": 1,
+              "OreDict:8": ""
+            },
+            "2:10": {
+              "id:8": "TConstruct:jerky",
+              "Count:3": 8,
+              "Damage:2": 2,
+              "OreDict:8": ""
+            },
+            "3:10": {
+              "id:8": "TConstruct:jerky",
+              "Count:3": 8,
+              "Damage:2": 3,
+              "OreDict:8": ""
+            },
+            "4:10": {
+              "id:8": "TConstruct:jerky",
+              "Count:3": 8,
+              "Damage:2": 4,
+              "OreDict:8": ""
+            }
+          }
+        },
+        "1:10": {
+          "rewardID:8": "bq_standard:item",
+          "index:3": 1,
+          "rewards:9": {
+            "0:10": {
+              "id:8": "dreamcraft:item.CoinCook",
+              "Count:3": 15,
+              "Damage:2": 0,
+              "OreDict:8": ""
+            }
+          }
+        }
+      }
+    },
+    "2899:10": {
+      "questID:3": 2899,
+      "preRequisites:11": [
+        2900
+      ],
+      "properties:10": {
+        "betterquesting:10": {
+          "snd_complete:8": "random.levelup",
+          "taskLogic:8": "AND",
+          "partySingleReward:1": 0,
+          "visibility:8": "NORMAL",
+          "isMain:1": 0,
+          "simultaneous:1": 0,
+          "icon:10": {
+            "id:8": "BiomesOPlenty:flippers",
+            "Count:3": 1,
+            "Damage:2": 0,
+            "OreDict:8": ""
+          },
+          "snd_update:8": "random.levelup",
+          "repeatTime:3": -1,
+          "globalShare:1": 1,
+          "questLogic:8": "AND",
+          "repeat_relative:1": 1,
+          "name:8": "Flippers",
+          "isGlobal:1": 0,
+          "lockedProgress:1": 1,
+          "autoClaim:1": 0,
+          "isSilent:1": 0,
+          "desc:8": "Maybe you\u0027re lucky and found some Flippers in a loot chest or dungeon.\nFlippers allow the player to swim at the same speed as walking.\nThe only problem is that it has 0 armor protection."
```

Comment:
Were flippers actually findable? I don't remember finding any. You might also want to mention that the underwater breathing potion makes it easier to see underwater? Or the night vision one. Or both, test it out.

Reply:
guess they in dungeons

Reply:
Night vision makes it bright, while water breathing gets rid of the fog and lets you breathe.

Reply:
sure if you found night vision at this point good for you. This quest is only a side quest